tests/test_apps/interpolation_to_hydra_config/my_app.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from omegaconf import DictConfig
+
+import hydra
+
+
+@hydra.main(config_path=".", config_name="config")
+def my_app(cfg: DictConfig) -> None:
+    pass
```

Comment:
Can you reuse simple_app here?
You can place the config next to it or somewhere else and override the used config like:
```
python tests/test_apps/simple_app/my_app.py  --config-name=config --config-dir=tests/test_apps/app_with_cfg
```

Reply:
Done in https://github.com/facebookresearch/hydra/pull/1696/commits/1d7bdd20fc4928af6b86d5c9b0f20625afbc9497.